torchbox/wagtail-torchbox
# Remove 'heading for processes' field on processes block

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/views/4019870)

### Description of Changes Made
Simply removes the field (which is not displayed anywhere in the front-end) and creates a migration to delete the content.
This is based off https://github.com/torchbox/wagtail-torchbox/pull/549 to avoid any migration clashes, so that will need to be deployed first, or they can be deployed together.

### How to Test
Edit a sub-proposition page, add a process block, and ensure you can't see a field called 'Heading for processes'.

### Screenshots

<details>
  <summary>Expand to see more</summary>

Before:

![Screenshot 2023-11-07 at 12 12 28](https://github.com/torchbox/wagtail-torchbox/assets/771869/b66a9fa3-688e-4d04-92f3-544f0111df79)

After:
![Screenshot 2023-11-07 at 12 12 39](https://github.com/torchbox/wagtail-torchbox/assets/771869/249e013c-8491-41ce-8bfd-70f82aa80753)


</details>

### MR Checklist
